packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -148,23 +146,92 @@ const RelationModalBody = ({ id }: RelationModalBodyProps) => {
   }
 
   const documentTitle = documentResponse.getTitle(documentLayoutResponse.edit.settings.mainField);
-
   const hasDraftAndPublished = documentResponse.schema?.options?.draftAndPublish ?? false;
 
+  const props = {
+    activeTab: 'draft',
+    collectionType: documentMeta.collectionType,
+    model: documentMeta.model,
+    documentId: documentMeta.documentId,
+    document: documentResponse.document,
+    meta: documentResponse.meta,
+  } satisfies DocumentActionProps;
+
   return (
     <Modal.Body>
       <DocumentRBAC permissions={permissions} model={documentMeta.model}>
-        <Flex direction="column" alignItems="flex-start" gap={2}>
-          <Typography tag="h2" variant="alpha">
-            {documentTitle}
-          </Typography>
-          {hasDraftAndPublished ? (
-            <Box marginTop={1}>
-              <DocumentStatus status={documentResponse.document?.status} />
-            </Box>
-          ) : null}
-        </Flex>
-        <FormContext initialValues={initialValues} method={id ? 'PUT' : 'POST'}>
+        <FormContext
+          initialValues={initialValues}
+          validate={(values: Record<string, unknown>, options: Record<string, string>) => {
+            const yupSchema = createYupSchema(
+              documentResponse.schema?.attributes,
+              documentResponse.components,
+              {
+                status: documentResponse.document?.status,
+                ...options,
+              }
+            );
+
+            return yupSchema.validate(values, { abortEarly: false });
+          }}
+          method="PUT"
```

Comment:
maybe you can add a TODO to change this value when we introduce the creation modal